.travis.yml

```diff
@@ -1,8 +1,5 @@
-branches:
-  only:
-    - master
 language: node_js
 node_js:
-  - "node"
```

Comment:
Can we add this back in?  It should test the latest version.  Also, any reasons why you just used 8 and 12?  Based on https://nodejs.org/en/about/releases/, we should probably support 10, 12, 14 and latest _(just to catch any new releases before the next TravisCI configuration change)_.

Reply:
Tests fail on Node 6 because some build tools don't support it. Node 6 has been EOL since April 2019, so I think it can be left out. I'll add versions 10 and 14.

Reply:
Is it possible to enable CI for PRs?

Reply:
Is `node` a label for Node 6?  If so, I didn't realize that.  So long as we support the versions mentioned above, we should be good.

Reply:
I thought so, but I seem to be wrong, `node` points to the latest stable release, I'll add it back in.
Also `sudo: false` has no effect anymore, I deleted it.

Reply:
Enabling CI for PRs is probably here: https://travis-ci.org/github/whitlockjc/path-loader/settings